spec/spec.md

```diff
@@ -516,42 +516,130 @@ The following two serviceEndpoints MUST be supported in the DID Document, but on
+
+#### DID-Relative URL Structure
+An Issuer of a VC may have an “IdentityHub” node serviceEndpoint in the Issuer's DID Document. Identity Hub nodes are the single endpoint to look up objects associated with a DID, as defined in [[def: Identity Hub (0.0.1 Predraft)]] (Decentralized Web Node v0.0.1 predraft)). 
+The following process defines how a DID-Relative URL is composed:
+1.	Let the base URI authority portion of the DID URL string be the target DID being addressed.
+2.	Append a service parameter to the DID URL string with the value IdentityHub.
+3.	Assemble an array of the Message Descriptor objects are desired for encoding in the DID-relative URL
+4.	JSON stringify the array of Message Descriptor objects from Step 3, then Base64Url encode the stringified output.
+5.	Append a queries parameter to the DID URL string with the value set to the JSON stringified, Base64Url encoded output of Step 4.
+
+##### DID-relative URLs are composed of the following segments
+```json
+did:example:123 + ?service=IdentityHub + &queries= + toBase64Url( JSON.stringify( [{ DESCRIPTOR_1 }, { DESCRIPTOR_N }] ) )
+did:example:123?service=IdentityHub&queries=W3sgTUVTU0FHRV8xIH0sIHsgTUVTU0FHRV9OIH1d..
+```
+The **queries* parameter value is neither JSON stringified nor Base64Url encoded, but should be when using Identity Hub URLs in practice 
+```json
+did:example:123?service=IdentityHub&queries=[{ "method": "CollectionsQuery", "schema": "https://schema.org/SocialMediaPosting" }]
+```
+```json
+did:example:123?service=IdentityHub&queries=W3sgTUVTU0FHRV8xIH0sIHsgTUVTU0FHRV9OIH1d...
+```
+**Resolve DID to locate the Identity Hub URIs:**
+```json
+did:example:123 --> resolve to Identity Hub endpoint(s) --> https://hub.example.com/
+```
+Below is a non-normative example of a DID Document that includes a serviceEndpoint:
+```json
+"service": [
+        "id": "#hub",
+        "type": "IdentityHub",
+        "serviceEndpoint": {
+          "instances": [
+            "https://beta.hub.msidentity.com/v1.0/a492cff2-d733-4057-95a5-a71fc3695bc8"  ],
+          "origins": []
+        }
+      }
+    ],
+```
+Using the "CollectionsQuery" method and the message descriptor as "objectId", encodedlist can be queried. "objectId" is the "id" field in credentialstatus.
 
-The issued VC MAY include a `credentialStatus` property
+Below is a non-normative example of credentialStatus property defined in VC: 
+```json
+"credentialStatus": {	
+  "id": "Qmdfr32sdf32546...",
+  "type": "StatusList2021",
```

Comment:
Shouldn't it be `StatusList2021` according to https://w3c-ccg.github.io/vc-status-list-2021/#statuslist2021entry ?

Reply:
I think we agreed to use this commit ver of StatusList2021 in v0.01 of this profile: https://github.com/w3c-ccg/vc-status-list-2021/tree/343b8b59cddba4525e1ef355356ae760fc75904e

so it should be RevocationList2021Status:
https://github.com/w3c-ccg/vc-status-list-2021/blob/343b8b59cddba4525e1ef355356ae760fc75904e/index.html#L252

when we move to final StatusList2021 in v1 of this profile, it will be StatusList2021Entry